stdpopsim/cli.py

```diff
@@ -293,6 +293,8 @@ def write_citations(engine, model, contig, species):
     """
     cite_str = ["If you use this simulation in published work, please cite:"]
     for citation in get_citations(engine, model, contig, species):
+        if "mutation rate" in citation.reasons and model.mutation_rate is not None:
```

Comment:
We have constants for this I think, don't need to use a hard coded string. citations.CiteReason.MUT_RATE I think?